src/generated/runtime.ts

```diff
@@ -1,266 +1,211 @@
 // tslint:disable
 /**
- * OH 2.0 Api Documentation
- * OH 2.0 Api Documentation
- *
- * The version of the OpenAPI document: 1.0
+ * Open Hospital API Documentation
+ * Open Hospital API Documentation
  *
+ * The version of the OpenAPI document: 0.1.0
+ * 
  *
  * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
  * https://openapi-generator.tech
  * Do not edit the class manually.
  */
 
-import { Observable, of, Subscriber } from "rxjs";
-import { ajax, AjaxRequest, AjaxResponse } from "rxjs/ajax";
-import { map, concatMap } from "rxjs/operators";
+import { Observable, of, Subscriber } from 'rxjs';
+import { ajax, AjaxRequest, AjaxResponse } from 'rxjs/ajax';
+import { map, concatMap } from 'rxjs/operators';
 
-export const BASE_PATH = "https://oh2.open-hospital.org".replace(/\/+$/, "");
+export const BASE_PATH = 'http://localhost:8080'.replace(/\/+$/, '');
```

Comment:
This must remain unchanged @SilverD3 ?

Reply:
Yes. See this [comment](https://github.com/informatici/openhospital-ui/pull/662#discussion_r1766295038),